src/south/OPCHDA/OPCHDA.class.js

```diff
@@ -223,7 +235,7 @@ class OPCHDA extends ProtocolHandler {
       }
     } catch (error) {
       this.logger.error(error)
-      this.logger.debug(`Agent stdout error: ${log}`)
+      this.logger.debug(`HDA Agent stdout error: ${log}`)
```

Comment:
we should probably add one error log rather than splitting this into two parts
this.logger.error('error occured when parsing log ${log}, error) or something like that